index.html

```diff
@@ -14,8 +14,436 @@
       href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/spoqa/spoqa-han-sans/css/SpoqaHanSansNeo.css">
+    <script src="https://code.iconify.design/iconify-icon/2.1.0/iconify-icon.min.js"></script>
     <script src="./main.js" async defer type="module"></script>
+
+    <style>
+      html {
+        font-size: 62.5%;
+      }
+
+      body {
+            font-family: 'Spoqa Han Sans Neo', sans-serif;
+            width : 80%;
+            margin-left:auto;
+            margin-right:auto;
+            max-width:130rem;
+      }
+
+      header {
+            margin: 2rem 0;
+            border-bottom: 0.1rem solid #D3D3D3;
+            padding-bottom: 1rem; 
+            display: flex; 
+            justify-content: space-between;
+      }
+
+      header img {
+        width: 4rem;
+        height: 2rem;
+      }
+
+      header span {
+        font-size : 2rem;
+        margin-left : 1rem;
+      }
+
+      header img, span {
+        vertical-align : middle;
+      }
+
+      
+      footer {
+        background-color: #EFEFEF; 
+        padding-bottom : 10rem;
+        padding-right : 10rem;
+        padding-left : 10rem;
+      }
+
+      footer > div {
+        display:flex;
+        justify-content: space-between; 
+        font-size:1.6rem;
+      }
```

Comment:
@SamTheKorean 님의 코드를 보았을 때, 일관적인 css 코드 작성 규칙이 따로 없어보이는데요. 띄어쓰기, 개행 여부 등 곳곳이 다 다르게 작성되어 있습니다.
특별히 지켜야 하는 규칙은 없습니다만 아래 처럼 작성하는 것이 일반적인 관습입니다.
일관적인 스타일로 코드를 작성하면, 코드 리뷰 시 중요한 변경에만 집중할 수 있습니다!

흔히 통용되는 JS 코딩 스타일 관습은 [airbnb의 룰](https://github.com/airbnb/javascript)인데요.
CSS나 HTML의 코딩 스타일 관습은 [google의 룰](https://github.com/airbnb/javascript)을 참고할 수 있겠습니다.

```suggestion
      footer {
        background-color: #EFEFEF; 
        padding-bottom: 10rem;
        padding-right: 10rem;
        padding-left: 10rem;
      }

      footer > div {
        display: flex;
        justify-content: space-between; 
        font-size: 1.6rem;
      }
```

Reply:
피드백 감사합니다! 제가 코딩 스타일 부분을 지나치게 간과했네요.. 공유해주신 레퍼런스 참조해서 다시 정리해보겠습니다1